ccd-definition/CaseEventToFields/CareSupervision/manageOrders/5_orderDetailsSection.json

```diff
@@ -250,12 +250,25 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "manageOrders",
-    "CaseFieldID": "manageOrdersIsByConsent",
+    "CaseFieldID": "manageOrdersC43Orders",
```

Comment:
On a call with Soorya. We are discussing possible name alternatives that might be more descriptive.
Some options:
- manageOrdersMultiSelectList
- manageOrdersMultiSelectListForC43